package.json

```diff
@@ -75,27 +68,31 @@
     "transform": {
       "^.+\\.(js|jsx)$": "<rootDir>/node_modules/babel-jest"
     },
-    "transformIgnorePatterns": [
-      "[/\\\\]node_modules[/\\\\].+\\.(js|jsx)$"
-    ],
+    "transformIgnorePatterns": ["[/\\\\]node_modules[/\\\\].+\\.(js|jsx)$"],
     "moduleNameMapper": {
       "^react-native$": "react-native-web"
     },
-    "moduleFileExtensions": [
-      "web.js",
-      "js",
-      "json",
-      "web.jsx",
-      "jsx",
-      "node"
-    ]
+    "moduleFileExtensions": ["web.js", "js", "json", "web.jsx", "jsx", "node"]
   },
   "babel": {
-    "presets": [
-      "react-app"
-    ]
+    "presets": ["react-app"]
   },
   "dependencies": {
-    "ascii-data-table": "^2.1.1"
+    "@data-ui/radial-chart": "0.0.11",
+    "@data-ui/theme": "0.0.9",
+    "@data-ui/xy-chart": "0.0.25",
+    "@vx/axis": "^0.0.142",
+    "@vx/glyph": "0.0.140",
+    "@vx/gradient": "^0.0.140",
+    "@vx/grid": "^0.0.142",
+    "@vx/group": "^0.0.140",
+    "@vx/legend": "0.0.143",
+    "@vx/mock-data": "^0.0.136",
+    "@vx/scale": "^0.0.143",
+    "@vx/shape": "0.0.142",
+    "ascii-data-table": "^2.1.1",
+    "neo4j-driver": "^1.4.1",
```

Comment:
Is the driver explicitly needed? If so, I think we'd rather have is as a `peerDependency` + `devDependency`.

Reply:
Removing. Not required.